pwnlib/elf/elf.py

```diff
@@ -1328,13 +1328,23 @@ def relro(self):
         .. _DF_BIND_NOW: http://refspecs.linuxbase.org/elf/gabi4+/ch5.dynamic.html#df_bind_now
 
         """
+        if not any('GNU_RELRO' in str(s.header.p_type) for s in self.segments):
```

Comment:
I uploaded some x86 binaries (I do not think arch really matters here, but I can upload x64 too if you'd like), tests are now running and all of these show the same (and imho correct) relro type as checksec.sh.